server/src/main/java/org/elasticsearch/repositories/RepositoriesStats.java

```diff
@@ -72,6 +103,39 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
             }
             builder.field("total_read_throttled_time_nanos", totalReadThrottledNanos);
             builder.field("total_write_throttled_time_nanos", totalWriteThrottledNanos);
+            if (shardSnapshotsStarted != -1) {
+                builder.field("shard_snapshots_started", shardSnapshotsStarted);
+            }
```

Comment:
better to have -1 represent not present than publishing zeroes here because zeroes are potentially signficant

Reply:
It should only matter in a BWC scenario anyway. Perhaps I'm overthinking it.